src/02 - Object types.md

```diff
@@ -454,6 +455,26 @@ The following notation is used in the tables below:
 |      |        |                        |          |             |           |  |      |      |       |      |               |                                                                                                                                                                        |           |
 |      | ref    | refRegistrering        |    x     |             |           |  |      |  x   |   x   |      |       x       | <p>&bull; references **Basisregistrering** or **Journalpost**</p><p>&bull; may be partially supported by systems</p><p>&bull; *unlink* not supported in Documaster</p> | &nbsp;    |
 
+### Dokumentflyt
```

Comment:
Fields `flytFra` and `flytFraBrukerIdent` are missing.

Reply:
Fixed. We had done internal model changes and I had not updated it.